applications/cmd_line/src/svk_quantify.cc

```diff
@@ -279,12 +295,9 @@ int main (int argc, char** argv)
             quantMets->LimitToSelectedVolume();
         }    
         quantMets->SetInput( reader->GetOutput() ); 
-        quantMets->LimitToSelectedVolume();    
```

Comment:
There is a conditional statement a few lines above. This line was overriding the conditional state set by the -b flag.